argobots.spec

```diff
@@ -104,10 +102,8 @@ rm -f %{buildroot}%{_libdir}/*.la
 %{_includedir}/*
 
 %changelog
-* Tue Feb 02 2021 B.Faccini <bruno.faccini@intel.com> - 1.1~a1-1.g7fd1987
-- upgrade to 1.1a1
-- add a patch to bring up 7fd1987 which enables libunwind support
-- build with unwinding enabled
+* Tue Feb 23 2021 B.Faccini <bruno.faccini@intel.com> - 1.1~b1-1
+- Update to 1.1b1
```

Comment:
Don't you want to keep info that unwinding has been configured/enabled ?

Reply:
Yes.  Thanks for the catch.

Done.